lib/final_destination.rb

```diff
@@ -114,7 +114,7 @@ def request_headers
       "User-Agent" => @user_agent,
       "Accept" => "*/*",
       "Accept-Language" => "*",
-      "Host" => @uri.hostname,
+      # "Host" => @uri.hostname,
```

Comment:
For some reason this was causing a 403 error when doing a GET for the presigned URL on minio:

> The request signature we calculated does not match the signature you provided. Check your key and signing method.

If I remove this and the other Host header below (for the 127.0.0.1 address) it works fine...